alf/examples/ac_target_navigation.gin

```diff
@@ -49,16 +44,16 @@ num_sentence_tiles=800
 import alf.networks.target_navigation
 
 target_navigation.get_actor_network.conv_layer_params=%conv_layer_params
-target_navigation.get_actor_network.activation_fn=@tf.keras.activations.softsign
-target_navigation.get_actor_network.kernel_initializer=@tf.keras.initializers.GlorotUniform()
+ActorDistributionRnnNetwork.activation_fn=@tf.keras.activations.softsign
```

Comment:
This cannot be just RnnNetwork because target_navigation.get_actor_network could create either regular or RnnNetwork agents, depending on the setup.